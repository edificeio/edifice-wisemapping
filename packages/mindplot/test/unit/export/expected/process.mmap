<?xml version="1.0" encoding="UTF-8"?><ap:Map xmlns:ap="http://schemas.mindjet.com/MindManager/Application/2003"><ap:OneTopic><ap:Topic OId="0W54nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="1W54nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="2W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Oakland/Berkeley"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="3W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="San Mateo"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="4W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Other North"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="5W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="San Francisco"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="6W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Santa Clara"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="7W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Marin/Napa/Solano"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Northern California"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="8W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Hawaii"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2654nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="2754nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="2854nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Orange County Eye and Transplant Bank"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2954nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="3054nezC90m8NYAi2fjQvw=="><ap:Text PlainText="In 2010, 2,500 referrals forwarded to OneLegacy"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                            "/></ap:Topic></ap:SubTopics><ap:Text PlainText="Northern California Transplant Bank"><ap:Font/></ap:Text><ap:Color FillColor="ff00ffd5"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic><ap:Topic OId="3154nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Doheny Eye and Tissue Transplant Bank"><ap:Font/></ap:Text><ap:Color FillColor="ff00ffd5"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/><ap:Hyperlink Url="http://www.dohenyeyebank.org/"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Transplant Bank International"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="3254nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="3354nezC90m8NYAi2fjQvw=="><ap:Text PlainText="In 2010, 11,828 referrals"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                            "/></ap:Topic></ap:SubTopics><ap:Text PlainText="OneLegacy"><ap:Font/></ap:Text><ap:Color FillColor="ff00ffd5"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic><ap:Topic OId="3454nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="3554nezC90m8NYAi2fjQvw=="><ap:Text PlainText="In 2010, 2,555 referrals"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                            "/></ap:Topic></ap:SubTopics><ap:Text PlainText="San Diego Eye Bank"><ap:Font/></ap:Text><ap:Color FillColor="ff00ffd5"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic><ap:Topic OId="3654nezC90m8NYAi2fjQvw=="><ap:Text PlainText="California Transplant Donor Network"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="3754nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="3854nezC90m8NYAi2fjQvw=="><ap:Text PlainText="In 2010, 0 referrals"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="California Transplant Services"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="3954nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Lifesharing"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="4054nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="4154nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="4254nezC90m8NYAi2fjQvw=="><ap:Text PlainText="In 2010, 2.023 referrals"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                            "/></ap:Topic></ap:SubTopics><ap:Text PlainText="Sierra Eye and Tissue Donor Services"><ap:Font/></ap:Text><ap:Color FillColor="ff00ffd5"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic></ap:SubTopics><ap:Text PlainText="DCI Donor Services"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="4354nezC90m8NYAi2fjQvw=="><ap:Text PlainText="SightLife"><ap:Font/></ap:Text><ap:Color FillColor="ff00ffd5"/><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/></ap:Topic></ap:SubTopics><ap:Text PlainText="Corneal Tissue OPS"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="9W54nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="1054nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Los Angeles"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1154nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Anaheim/Santa Ana"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1254nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Ventura"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1354nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Other South"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Southern California"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="4754nezC90m8NYAi2fjQvw=="><ap:Text PlainText="QE Medicare"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1454nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="1554nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="1654nezC90m8NYAi2fjQvw=="><ap:Text PlainText="AAO"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1754nezC90m8NYAi2fjQvw=="><ap:Text PlainText="ASCRS"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1854nezC90m8NYAi2fjQvw=="><ap:Text PlainText="EBAA"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Advocacy"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1954nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Military"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2054nezC90m8NYAi2fjQvw=="><ap:Text PlainText="United Network for Organ Sharing"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2154nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Kaiser Hospital System"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2254nezC90m8NYAi2fjQvw=="><ap:Text PlainText="University of California System"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2354nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="2454nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Medicare Part A"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2554nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Medicare Part B"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="CMS"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Policy Bodies"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="4854nezC90m8NYAi2fjQvw=="><ap:Text PlainText="CMS Data"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="4454nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="4554nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Darthmouth Atlas of Health"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="4654nezC90m8NYAi2fjQvw=="><ap:Text PlainText="HealthLandscape"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Tools"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="4954nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="5054nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="5154nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="5254nezC90m8NYAi2fjQvw=="><ap:Text PlainText="65780 (amniotic membrane tranplant"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="5354nezC90m8NYAi2fjQvw=="><ap:Text PlainText="65781 (limbal stem cell allograft)"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="5454nezC90m8NYAi2fjQvw=="><ap:Text PlainText="65782 (limbal conjunctiva autograft)"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Ocular Reconstruction Transplant"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="5554nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="5654nezC90m8NYAi2fjQvw=="><ap:Text PlainText="65756"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Endothelial keratoplasty"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="5754nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="5854nezC90m8NYAi2fjQvw=="><ap:Text PlainText="65767"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Epikeratoplasty"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="CPT's which don't allow V2785"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="5954nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="6054nezC90m8NYAi2fjQvw=="><ap:Text PlainText="65710"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Anterior lamellar keratoplasty"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="6154nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="6254nezC90m8NYAi2fjQvw=="><ap:Text PlainText="V2785"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="6354nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="6454nezC90m8NYAi2fjQvw=="><ap:Text PlainText="0290T"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Laser incision in recepient"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Processing, preserving, and transporting corneal tissue"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="6554nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="6654nezC90m8NYAi2fjQvw=="><ap:Text PlainText="0289T"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Laser incision in donor"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="6754nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="6854nezC90m8NYAi2fjQvw=="><ap:Text PlainText="65730 (in other)"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="6954nezC90m8NYAi2fjQvw=="><ap:Text PlainText="65755 (in pseudoaphakia)"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="7054nezC90m8NYAi2fjQvw=="><ap:Text PlainText="65750 (in aphakia)"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Penetrating keratoplasty"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Ambulatory Payment Classification"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="California"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/><ap:SubTopicsShape SubTopicsAlignment="urn:mindjet:Center" SubTopicsGrowth="urn:mindjet:Horizontal" SubTopicsGrowthDirection="urn:mindjet:AutomaticHorizontal&#10;                        " VerticalDistanceBetweenSiblings="150"/></ap:Topic></ap:OneTopic><ap:Relationships><ap:Relationship><ap:ConnectionGroup Index="0"><ap:Connection><ap:ObjectReference OIdRef="3254nezC90m8NYAi2fjQvw=="/></ap:Connection></ap:ConnectionGroup><ap:ConnectionGroup Index="1"><ap:Connection><ap:ObjectReference OIdRef="2754nezC90m8NYAi2fjQvw=="/></ap:Connection></ap:ConnectionGroup><ap:AutoRoute AutoRouting="true"/></ap:Relationship><ap:Relationship><ap:ConnectionGroup Index="0"><ap:Connection><ap:ObjectReference OIdRef="3654nezC90m8NYAi2fjQvw=="/></ap:Connection></ap:ConnectionGroup><ap:ConnectionGroup Index="1"><ap:Connection><ap:ObjectReference OIdRef="2754nezC90m8NYAi2fjQvw=="/></ap:Connection></ap:ConnectionGroup><ap:AutoRoute AutoRouting="true"/></ap:Relationship></ap:Relationships></ap:Map>