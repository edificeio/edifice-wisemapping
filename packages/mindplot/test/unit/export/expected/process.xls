<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<?mso-application progid="Excel.Sheet"?><Workbook xmlns:x="urn:schemas-microsoft-com:office:excel" xmlns:ss="urn:schemas-microsoft-com:office:spreadsheet" xmlns:duss="urn:schemas-microsoft-com:office:dummyspreadsheet" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns="urn:schemas-microsoft-com:office:spreadsheet">
<Styles>
<Style ss:ID="s16" ss:Name="attribute_cell">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
</Style>
<Style ss:ID="s17" ss:Name="attribute_header">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
<Font ss:Bold="1"/>
</Style>
</Styles>
<Worksheet ss:Name="FreeMind Sheet">
<Table>
<Row>
<Cell ss:Index="1">
<Data ss:Type="String">California</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Northern California</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Oakland/Berkeley</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">San Mateo</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Other North</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">San Francisco</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Santa Clara</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Marin/Napa/Solano</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Hawaii</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Corneal Tissue OPS</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Transplant Bank International</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Orange County Eye and Transplant Bank</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Northern California Transplant Bank</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="5">
<Data ss:Type="String">In 2010, 2,500 referrals forwarded to OneLegacy</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Doheny Eye and Tissue Transplant Bank</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">OneLegacy</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">In 2010, 11,828 referrals</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">San Diego Eye Bank</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">In 2010, 2,555 referrals</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">California Transplant Donor Network</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">California Transplant Services</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">In 2010, 0 referrals</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Lifesharing</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">DCI Donor Services</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Sierra Eye and Tissue Donor Services</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="5">
<Data ss:Type="String">In 2010, 2.023 referrals</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">SightLife</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Southern California</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Los Angeles</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Anaheim/Santa Ana</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Ventura</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Other South</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">QE Medicare</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Policy Bodies</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Advocacy</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">AAO</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">ASCRS</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">EBAA</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Military</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">United Network for Organ Sharing</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Kaiser Hospital System</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">University of California System</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">CMS</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Medicare Part A</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Medicare Part B</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">CMS Data</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Tools</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Darthmouth Atlas of Health</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">HealthLandscape</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Ambulatory Payment Classification</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">CPT's which don't allow V2785</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Ocular Reconstruction Transplant</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="5">
<Data ss:Type="String">65780 (amniotic membrane tranplant</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="5">
<Data ss:Type="String">65781 (limbal stem cell allograft)</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="5">
<Data ss:Type="String">65782 (limbal conjunctiva autograft)</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Endothelial keratoplasty</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="5">
<Data ss:Type="String">65756</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Epikeratoplasty</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="5">
<Data ss:Type="String">65767</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Anterior lamellar keratoplasty</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">65710</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Processing, preserving, and transporting corneal tissue</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">V2785</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Laser incision in recepient</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="5">
<Data ss:Type="String">0290T</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Laser incision in donor</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">0289T</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Penetrating keratoplasty</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">65730 (in other)</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">65755 (in pseudoaphakia)</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">65750 (in aphakia)</Data>
</Cell>
</Row>
</Table>
</Worksheet>
</Workbook>
