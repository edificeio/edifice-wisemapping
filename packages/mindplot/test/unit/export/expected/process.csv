California 
,
            Northern California 
,
            ,
            Oakland/Berkeley 
,
            ,
            San Mateo 
,
            ,
            Other North 
,
            ,
            San Francisco 
,
            ,
            Santa Clara 
,
            ,
            Marin/Napa/Solano 
,
            Hawaii 
,
            Corneal Tissue OPS 
,
            ,
            Transplant Bank International 
,
            ,
            ,
            Orange County Eye and Transplant Bank 
,
            ,
            ,
            Northern California Transplant Bank 
,
            ,
            ,
            ,
            In 2010, 2,500 referrals forwarded to OneLegacy 
,
            ,
            ,
            Doheny Eye and Tissue Transplant Bank 
,
            ,
            OneLegacy 
,
            ,
            ,
            In 2010, 11,828 referrals 
,
            ,
            San Diego Eye Bank 
,
            ,
            ,
            In 2010, 2,555 referrals 
,
            ,
            California Transplant Donor Network 
,
            ,
            California Transplant Services 
,
            ,
            ,
            In 2010, 0 referrals 
,
            ,
            Lifesharing 
,
            ,
            DCI Donor Services 
,
            ,
            ,
            Sierra Eye and Tissue Donor Services 
,
            ,
            ,
            ,
            In 2010, 2.023 referrals 
,
            ,
            SightLife 
,
            Southern California 
,
            ,
            Los Angeles 
,
            ,
            Anaheim/Santa Ana 
,
            ,
            Ventura 
,
            ,
            Other South 
,
            QE Medicare 
,
            Policy Bodies 
,
            ,
            Advocacy 
,
            ,
            ,
            AAO 
,
            ,
            ,
            ASCRS 
,
            ,
            ,
            EBAA 
,
            ,
            Military 
,
            ,
            United Network for Organ Sharing 
,
            ,
            Kaiser Hospital System 
,
            ,
            University of California System 
,
            ,
            CMS 
,
            ,
            ,
            Medicare Part A 
,
            ,
            ,
            Medicare Part B 
,
            CMS Data 
,
            Tools 
,
            ,
            Darthmouth Atlas of Health 
,
            ,
            HealthLandscape 
,
            Ambulatory Payment Classification 
,
            ,
            CPT's which don't allow V2785 
,
            ,
            ,
            Ocular Reconstruction Transplant 
,
            ,
            ,
            ,
            65780 (amniotic membrane tranplant 
,
            ,
            ,
            ,
            65781 (limbal stem cell allograft) 
,
            ,
            ,
            ,
            65782 (limbal conjunctiva autograft) 
,
            ,
            ,
            Endothelial keratoplasty 
,
            ,
            ,
            ,
            65756 
,
            ,
            ,
            Epikeratoplasty 
,
            ,
            ,
            ,
            65767 
,
            ,
            Anterior lamellar keratoplasty 
,
            ,
            ,
            65710 
,
            ,
            Processing, preserving, and transporting corneal tissue 
,
            ,
            ,
            V2785 
,
            ,
            ,
            Laser incision in recepient 
,
            ,
            ,
            ,
            0290T 
,
            ,
            Laser incision in donor 
,
            ,
            ,
            0289T 
,
            ,
            Penetrating keratoplasty 
,
            ,
            ,
            65730 (in other) 
,
            ,
            ,
            65755 (in pseudoaphakia) 
,
            ,
            ,
            65750 (in aphakia) 
